ui/DesignSystem/Component/Badge.svelte

```diff
@@ -0,0 +1,17 @@
+<script>
+  export let variant = null;
+  export let style = null;
+
+  const caption = String(variant);
+</script>
+
+<style>
+  .badge {
+    color: var(--color-primary);
```

Comment:
The color should be `--color-secondary` and background should be `--color-secondary-level-1` according to Figma.
![image](https://user-images.githubusercontent.com/4406983/91294203-72c8db80-e799-11ea-8e90-21dbe085c394.png)